src/components/dashboard/charts/trafficChart.config.ts

```
@@ -0,0 +1,169 @@
+
+// x축에 표시할 시간 목록
+const LABEL_HOURS = new Set([0, 6, 12, 18, 24]);
+
+export const BASE_OPTIONS: ApexOptions = {
+  chart: {
+    id: CHART_ID,
+    type: "area",
+    events: {
+      // SVG <title> 및 canvas 폴백 텍스트 제거
+      mounted: (chartContext: { el: Element }) => {
+        chartContext.el.querySelectorAll("title").forEach((el) => el.remove());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)